docs/components/checkbox-field/demo/base-demo.md

```diff
@@ -3,7 +3,10 @@
   @label='Label'
   @isChecked={{this.isChecked}}
   @onChange={{this.handleChange}}
-/>
+>
+  <:label>extra label info</:label>
```

Comment:
Was not feeling super imaginative here...if anyone has any suggestions of what to put in the blocks I will do a sweep and update the content in a single commit!

Reply:
You could maybe do something like what we are expecting to do - add an icon here (with no tooltip for now):

```html
<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" stroke="currentColor" viewBox="0 0 24 24"><path d="M12 3a9 9 0 11-6.364 2.636A8.972 8.972 0 0112 3zm0 4.7v5.2m0 3.39v.01" fill="none" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"></path></svg>
```

Reply:
oh wait...with these changes all these `:named-blocks` should be removed from these demos.